financial-services-accelerator/components/org.wso2.financial.services.accelerator.common/src/test/java/org/wso2/financial/services/accelerator/common/test/FSConfigParserTests.java

```diff
@@ -68,16 +69,16 @@ public void testConfigParserInit() {
         Assert.assertEquals(openBankingConfigParser.getConfiguration().get("Sample.FSHandler"), "DummyValue");
         Assert.assertEquals(openBankingConfigParser.getConfiguration().get("Sample.FSHandler2"), "property.value");
         Assert.assertNotNull(openBankingConfigParser.getConfiguration().get("Sample.FSHandler4"));
-        Map<String, Map<Integer, String>> openBankingExecutors =
-                FinancialServicesConfigParser.getInstance().getFinancialServicesExecutors();
+        Map<String, Map<Integer, String>> openBankingExecutors = FinancialServicesConfigParser.getInstance()
```

Comment:
here as well

Reply:
FIxed in https://github.com/wso2/financial-services-accelerator/pull/110/commits/fe1e2b6aa311de998ac4f0028ad72f9247c4dd0c